sdks/typescript/__tests__/__mocks__/@honcho-ai/core.ts

```
@@ -0,0 +1,45 @@
+      getContext: jest.fn(),
+      search: jest.fn(),
+    },
+    getOrCreate: jest.fn().mockResolvedValue({ id: 'test-workspace', metadata: {} }),
+    update: jest.fn(),
+    list: jest.fn(),
+    search: jest.fn(),
+  };
+
+  constructor(options?: any) {
+    // Mock constructor
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)